client/my-sites/plans/jetpack-plans/product-lightbox/style.scss

```
@@ -132,7 +132,7 @@
 
 .product-lightbox__detail-list {
 	p {
-		font-size: 0.875rem;
+		font-size: 1rem;
```

Now since, we have it `1rem`, let us re-use the variable
```suggestion
		font-size: $font-body;
```